webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -24,6 +25,12 @@ export const createInitialTaxonomy = (currentTaxonomy, availableTaxonomies) => {
 export const getCurrentPath = () =>
   removeLastSlashFromPath(window.location.pathname);
 
+export const cleanNavPath = href => {
+  if (!href) return href;
+  const uri = new URI(href);
```

Comment:
I was wondering why `URI` and not [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL). Answering my own question: `new URL(href)` will fail if it's just a path. Reading https://medialize.github.io/URI.js/ it can handle paths